### PR TITLE
@eessex => [Tooltips] adds slight delay on hiding tooltips

### PR DESCRIPTION
**Pull request by l2succes**
This isn't perfect yet, we're gonna need to keep tweaking the logic on when to show/hide tooltips but it's better than where we are now.

### CODE REVIEW

**Comment by eessex** [comment]
Possible to show a gif of this interaction?  Looks ok to be besides the ❌

**Comment by eessex** [comment]
This should also include a release tag

File: src/Components/OverlayTrigger/index.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export class OverlayTrigger extends React.Component<OverlayTriggerProps> {
           {children}
         </Container>
 
-        <Overlay target={this.target} {...props}>
+        <Overlay {...props} target={() => findDOMNode(this.target)}>
           {cloneElement(overlay, {
```
Comment (eessex) [inline src/Components/OverlayTrigger/index.tsx]: Is this related to a refs complaint?

File: src/Components/Publishing/ToolTip/LinkWithTooltip.tsx [hunk 2]
```diff
@@ -50,32 +52,34 @@ export class LinkWithTooltip extends Component<Props, State> {
...
       <div />
     )
 
     const { show } = this.state
     return (
-      <OverlayTrigger show={show} placement="top" overlay={toolTip}>
+      <OverlayTrigger
+        show={show}
+        placement="top"
+        // container={this}
```
Comment (eessex) [inline src/Components/Publishing/ToolTip/LinkWithTooltip.tsx]: stray comment or intentional?

File: src/Components/Publishing/ToolTip/ToolTip.tsx [hunk 3]
```diff
@@ -36,16 +36,21 @@ export class ToolTip extends React.Component<Props> {
     if (!this.props.entity) return null
 
     return (
-      <ToolTipContainer>
-        {this.getToolTip()}
-        <ArrowDown />
+      <ToolTipContainer style={{ ...this.props.style }}>
```
Comment (eessex) [inline src/Components/Publishing/ToolTip/ToolTip.tsx]: Where is props.style coming from?

File: src/Components/Publishing/ToolTip/LinkWithTooltip.tsx (hunk 2 above)
Comment (l2succes) [reply src/Components/Publishing/ToolTip/LinkWithTooltip.tsx]: it's a stray comment from experimenting

**Comment by l2succes** [comment]
@eessex I'm still working out some kinks to make this better, will ping you when it's ready

**Comment by l2succes** [comment]
@eessex Ok I think it's good enough for now

File: src/Components/OverlayTrigger/index.tsx (hunk 1 above)
Comment (l2succes) [reply src/Components/OverlayTrigger/index.tsx]: Yes, I believe that's where that error was coming from.
